tests/sentry/api/endpoints/test_group_attachments.py

```diff
@@ -66,31 +66,34 @@ def test_filter(self):
         assert len(response.data) == 1
         assert response.data[0]["id"] == str(attachment2.id)
 
-    def test_screenshot_filter(self):
+    def test_screenshot_across_groups(self):
         self.login_as(user=self.user)
 
-        attachment1 = self.create_attachment(type="event.attachment", file_name="screenshot.png")
-        self.create_attachment(type="event.attachment", file_name="screenshot-not.png")
-
-        with self.feature("organizations:event-attachments"):
-            response = self.client.get(self.path(screenshot=True))
-
-        assert response.status_code == 200, response.content
-        assert len(response.data) == 1
-        assert response.data[0]["id"] == str(attachment1.id)
-
-    def test_second_screenshot_filter(self):
-        self.login_as(user=self.user)
-
-        attachment1 = self.create_attachment(type="event.attachment", file_name="screenshot.png")
-        self.create_attachment(type="event.attachment", file_name="screenshot-not.png")
+        min_ago = before_now(minutes=1).isoformat()
+        group1_event = self.store_event(
+            data={"fingerprint": ["group1"], "timestamp": min_ago}, project_id=self.project.id
+        )
+        self.create_attachment(file_name="screenshot.png", event_id=group1_event.event_id)
+        self.create_attachment(file_name="screenshot-1.png", event_id=group1_event.event_id)
+        self.create_attachment(file_name="foo.png", event_id=group1_event.event_id)
```

Comment:
```suggestion
        # This will not be included as name doesn't contain 'screenshot'
        self.create_attachment(file_name="foo.png", event_id=group1_event.event_id)
```